packages/back-end/src/models/ApiKeyModel.ts

```diff
@@ -17,13 +18,29 @@ const apiKeySchema = new mongoose.Schema({
   description: String,
   organization: String,
   dateCreated: Date,
+  encryptSDK: Boolean,
+  encryptionPrivateKey: String,
   secret: Boolean,
 });
 
 type ApiKeyDocument = mongoose.Document & ApiKeyInterface;
 
 const ApiKeyModel = mongoose.model<ApiKeyDocument>("ApiKey", apiKeySchema);
 
+async function generatePrivateKey(): Promise<string> {
```

Comment:
@jdorn After updating this function, it felt like it made more sense to just house this directly in this file, rather than having the `subtle-crypto` util file.